src/CLI/Utilities/ArgumentHandler.cs

```diff
@@ -9,12 +10,88 @@ namespace CLI.Utilities
 {
     internal class ArgumentHandler
     {
-        internal async Task RunScripts(ArgumentParser argumentParser)
+        public RootCommand BuildRootCommand()
         {
+            var singleBoxDeployOption = new Option<bool>(
+            name: "--single-box-deploy",
+            getDefaultValue: () => false,
+            description: "Prepares and installs hub and spoke environments"
```

Comment:
[nit] move the arguments one indent in 😊

```suggestion
                name: "--single-box-deploy",
                getDefaultValue: () => false,
                description: "Prepares and installs hub and spoke environments"
```

Reply:
Also applies to below option definitions.